17/main.js

```diff
@@ -0,0 +1,113 @@
+const APIKey = "9SjprIauzm1kV2DrNv7UGEZL92uy576Z";
+const cityInputField = document.getElementById("name");
+const mainDiv = document.getElementById("main");
+const navbar = document.getElementById("getCityNeighbors-container");
+const childDiv = document.createElement("div");
+mainDiv.append(childDiv);
+const buttonCheckWeather = document.getElementById("addbutton");
+let counter = 0;
+function convertToCelsius(num) {
+  return ((num - 32) * 5) / 9;
+}
+async function getCityID(cityName) {
+  try {
+    const response = await fetch(
+      `http://dataservice.accuweather.com/locations/v1/cities/search?apikey=${APIKey}&q=${cityName}`
+    );
+    if (response.status === 200) {
```

Comment:
Fetch will always return 200, you should make sure that there is a result

<img width="539" alt="image" src="https://user-images.githubusercontent.com/84010501/204129878-013cfb73-2f49-480b-a0ea-6446af205d20.png">